.github/workflows/build-ios-tvos.yml

```diff
@@ -23,6 +23,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Download libs
       run: ./scripts/${{matrix.cfg.libs}}/download_libs.sh
+    - name: Download tvos libs
+      run: ./scripts/${{matrix.cfg.libs}}/download_libs_tvos.sh
```

Comment:
Maybe we should just do for both steps?:
```./scripts/${{matrix.cfg.target}}/download_libs.sh```

And have the two scripts both be `download_libs.sh` and live in `scripts/ios/` `scripts/tvos/` ?